Lib/test/test_asyncio/test_unix_events.py

```diff
@@ -1195,8 +1195,7 @@ async def test_fork_not_share_event_loop(self):
         if pid == 0:
             # child
             try:
-                with self.assertWarns(DeprecationWarning):
-                    loop = asyncio.get_event_loop_policy().get_event_loop()
+                loop = asyncio.get_event_loop()
```

Comment:
It was `asyncio.get_event_loop_policy().get_event_loop()`.

Reply:
using `get_event_loop` is more appropriate here,  it will test both that there is no running loop nor set loop